src/dotnet/CarbonAware.Plugins.NewBasicJsonPlugin/CarbonAwareJsonReaderPlugin.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections;
+using CarbonAware.Model;
+using CarbonAware.Plugins.BasicJsonPlugin;
+using Microsoft.Extensions.Logging;
+
+namespace CarbonAware.Plugins.NewBasicJsonPlugin;
```

Comment:
Need to rename the namespace for something more meaningful. Same as the project.